Stripe/STPBankAccountParams+Private.h

```diff
@@ -15,6 +15,7 @@ NS_ASSUME_NONNULL_BEGIN
 + (STPBankAccountHolderType)accountHolderTypeFromString:(NSString *)string;
 + (NSString *)stringFromAccountHolderType:(STPBankAccountHolderType)accountHolderType;
 
+- (instancetype)init;
```

Comment:
I think I was getting at making sure we have an explicit designated initializer for everything and seeing if that provides any compiler safety but didn't finish it. I'll remove this for now and restart the investigation later.